src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,98 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
-
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    
+    private Node<K, V>[] table;
+    private int threshold;
+    private int size;
+    
+    @SuppressWarnings("unchecked")
```

Comment:
"Bad practice", consider removing this annotation))

Reply:
Why? In JDK used in many places